plugins/woocommerce-admin/webpack.config.js

```diff
@@ -168,6 +168,19 @@ const webpackConfig = {
 		},
 	},
 	plugins: [
+		new CopyWebpackPlugin( {
+			patterns: [
+				{
+					from: path.resolve(
+						__dirname,
+						'bin/modified-editsite-lock-unlock.js'
+					), // Replace with the path to your custom script
+					to: require.resolve(
+						'@wordpress/edit-site/build-module/lock-unlock.js'
+					), // Replace with the destination path
+				},
+			],
+		} ),
```

Comment:
Cool approach. A highly visible comment explaining when this can be removed (ie, WP 6.5 when our L-2 support policy no longer qualifies WP 6.3) would be helpful here so we don't forget to remove it one day.

Reply:
Good suggestion!

Updated in https://github.com/woocommerce/woocommerce/pull/40884/commits/630c78a9ef9948926027212e4a764070140509d8